addons/netfox/serialization/value-to-bytes.gd

```diff
@@ -0,0 +1,202 @@
+extends Node
+class_name ValueToBytes ##Mayhaps needs to become Autoload so as to use the logger.
```

Comment:
I am *this* close to making a godotengine pull request for this (basically static constant which returns the byte size of each variant), but even if I did, it would take a while for merge, and further for netfox to update to latest version.

Anyhow, this class is required for all sorts of serialization, since the rollback properties of netfox are dynamic. The operations here are as cheap as they can be too. Except I guess the function serialize where I could cache the returned objects once they are once constructed. Anyway, encoding 3 floats (Vector3) shouldn't be noticeable.